Stopped working

Mine, seems to have stopped working. When I click the button nothing happens.

Okay I have a fix to use `rootUri` instead of `_rootUri` and that works in the sandbox.

I have an issue getting the extension to install in VS Code though - my first time installing it using WSL on Windows.
```
 DONE  Packaged: build/auto-commit-msg-0.25.1.vsix (26 files, 34.73KB)
<3>init: (20977) ERROR: UtilConnectToInteropServer:307: connect failed 2
                                                                        make: *** [Makefile:52: e] Error 1
```

--- a/docs/development/advanced/sandbox.md
+++ b/docs/development/advanced/sandbox.md
@@ -9,7 +9,7 @@ If you prefer not use this approach, you can just compile and install the extens
 
 ## Start sandbox
 
-These are configured in the `.vscode` directory of the repo.
+These are configured in the `.vscode/launch.json` config.
 
 Follow these steps:
 
@@ -25,7 +25,7 @@ Follow these steps:
         - NB. You'll must run `npm run sb` **first** to ensure this directory exists and then run the debug action. Run that NPM command again whenever you want to clear the space and start over.
 1. Click the run button.
 
-That will start a new sandboxed VS Code session which has the extension built using the local code and _enabled_, and all other extensions _disabled_. At a lower level, it runs `npm compile` and `npm watch`.
+That will start a new sandboxed VS Code session which has the extension built using the local code and _enabled_, and all other extensions _disabled_. At a lower level, it runs `npm compile` and `npm watch`. If you want to keep extensions enabled, remove the `--disable-extensions` flag in the launch config.
 
 What is especially useful about this is that whenever an extension action is performed in the sandboxed VS Code window, if there are any logs for that then those will appear in the Debug Console of the _original_ VS Code window.
 

--- a/src/extension.ts
+++ b/src/extension.ts
@@ -27,20 +27,18 @@ function _validateFoundRepos(git: API) {
 }
 
 /**
- * Run autofill against one of multiples in the workspace.
- *
- * This is a rare flow.
+ * Run autofill against one of multiples in the workspace or when using GitLens on a single repo.
  *
  * @param sourceControl Of type `vscode.SourceControl` with public `.rootUri`
- *   and private `.rootUri`.
+ *   and private `._rootUri` (the latter is not set when using GitLens).
  */
 async function _handleRepos(git: API, sourceControl: any) {
   // FIXME: Unfortunately this seems to only pick up the first repo and not find
   // second, etc.
   const selectedRepo = git.repositories.find(repository => {
-    const uri = sourceControl._rootUri;
+    const uri = sourceControl.rootUri;
     if (!uri) {
-      console.warn("_rootUri not set");
+      console.warn("rootUri not set on VS Code source control");
     }
     return repository.rootUri.path === uri.path;
   });

--- a/src/extension.ts
+++ b/src/extension.ts
@@ -28,17 +28,16 @@ function _validateFoundRepos(git: API) {
 
 /**
  * Run autofill against one of multiples in the workspace or when using GitLens on a single repo.
- *
- * @param sourceControl Of type `vscode.SourceControl` with public `.rootUri`
- *   and private `._rootUri` (the latter is not set when using GitLens).
  */
-async function _handleRepos(git: API, sourceControl: any) {
+async function _handleRepos(git: API, sourceControl: vscode.SourceControl) {
   // FIXME: Unfortunately this seems to only pick up the first repo and not find
-  // second, etc.
+  // second, etc. If you repository through to makeAndFillCommitMsg, getChanges
+  // and diffIndex etc. and replace "getWorkspaceFolder" usage then that will work.
   const selectedRepo = git.repositories.find(repository => {
     const uri = sourceControl.rootUri;
     if (!uri) {
-      console.warn("rootUri not set on VS Code source control");
+      console.warn("rootUri not set for current repo");
+      return false;
     }
     return repository.rootUri.path === uri.path;
   });
@@ -61,14 +60,14 @@ async function _handleRepo(git: API) {
 /**
  * Choose the relevant repo and apply autofill logic on files there.
  */
-async function _chooseRepoForAutofill(uri?: vscode.Uri) {
+async function _chooseRepoForAutofill(sourceControl?: vscode.SourceControl) {
   const git = getGitExtension()!;
   _validateFoundRepos(git);
 
   vscode.commands.executeCommand("workbench.view.scm");
 
-  if (uri) {
-    _handleRepos(git, uri);
+  if (sourceControl) {
+    _handleRepos(git, sourceControl);
   } else {
     _handleRepo(git);
   }

